bftengine/include/bcstatetransfer/SimpleBCStateTransfer.hpp

```diff
@@ -106,28 +107,46 @@ struct Config {
   bool pedanticChecks = false;
   bool isReadOnly = false;
 
-#if defined USE_COMM_PLAIN_TCP || defined USE_COMM_TLS_TCP
-  uint32_t maxChunkSize = 16384;
-  uint16_t maxNumberOfChunksInBatch = 8;
-#else
-  // maxChunkSize * maxNumberOfChunksInBatch should not exceed 64KB as UDP message size is limited to 64KB.
-  uint32_t maxChunkSize = 2048;
-  uint16_t maxNumberOfChunksInBatch = 32;
-#endif
-
-  uint32_t maxBlockSize = 10 * 1024 * 1024;                      // 10MB
-  uint32_t maxPendingDataFromSourceReplica = 256 * 1024 * 1024;  // Maximal internal buffer size for all ST data
-  uint32_t maxNumOfReservedPages = 2048;
-  uint32_t sizeOfReservedPage = 4096;
-
-  uint32_t refreshTimerMilli = 300;                               // ms
-  uint32_t checkpointSummariesRetransmissionTimeoutMilli = 2500;  // ms
-  uint32_t maxAcceptableMsgDelayMilli = 60000;                    // 1 minute
-  uint32_t sourceReplicaReplacementTimeoutMilli = 5000;           // 5 seconds
-  uint32_t fetchRetransmissionTimeoutMilli = 250;                 // ms
-  std::chrono::seconds metricsDumpIntervalSeconds;                // sec
+  // sizes
+  uint32_t maxChunkSize = 0;
+  uint16_t maxNumberOfChunksInBatch = 0;
+  uint32_t maxBlockSize = 0;                     // bytes
+  uint32_t maxPendingDataFromSourceReplica = 0;  // Maximal internal buffer size for all ST data, bytes
+  uint32_t maxNumOfReservedPages = 0;
+  uint32_t sizeOfReservedPage = 0;  // bytes
+
+  // timeouts
+  uint32_t refreshTimerMs = 0;
```

Comment:
Nit: Since we're already here, it might be a good opportunity to replace `uint32_t` with actual time types from `std::chrono`.

Reply:
agree, didn't want to mix joy and cheer in one commit